src/widgets/mobile-view/ui/mobile-view.tsx

```diff
@@ -20,106 +33,212 @@ const Divider = ({ color }: DividerProps) => <hr className={cx('divider', color)
-            onClick={onClose}
+            isDropdownActive={activeDropdowns.has(NAV_MENU_LABELS.MENTORSHIP)}
+            onMenuItemClick={() => onMenuItemClick(NAV_MENU_LABELS.MENTORSHIP)}
           />
-        ))}
-      </SchoolMenu>
-
-      <Divider color={color} />
-
-      <Link onClick={onClose} href={`/#${ANCHORS.DONATE}`} className={cx('category-link', color)}>
-        Donate
-      </Link>
 
-      <Divider color={color} />
+          <SchoolMenu isVisible={activeDropdowns.has(NAV_MENU_LABELS.MENTORSHIP)}>
+            <SchoolMenu.Item
+              key={NAV_MENU_LABELS.MENTORSHIP}
+              title="About Mentorship"
+              description="By teaching others, you learn yourself"
+              url={`/${ROUTES.MENTORSHIP}`}
+              color={color}
+              onClick={onClose}
+            />
+            {coursesWithMentorship.map((course) => (
+              <SchoolMenu.Item
+                key={course.id}
+                icon={course.iconSmall}
+                title={course.title}
+                description={course.startDate}
+                url={course.detailsUrl}
+                color={color}
+                onClick={onClose}
+              />
+            ))}
+          </SchoolMenu>
+        </div>
+
+        <Divider color={color} />
+
+        <Link onClick={onClose} href={`/${ROUTES.DOCS_EN}`} className={cx('category-link', color)}>
+          {NAV_MENU_LABELS.DOCS}
+        </Link>
+
+        <Divider color={color} />
+
+        <div className={cx('category-container')}>
+          <MobileNavItem
+            title={NAV_MENU_LABELS.SUPPORT_US}
+            icon={iconSrc}
+            color={color}
+            isDropdownActive={activeDropdowns.has(NAV_MENU_LABELS.SUPPORT_US)}
+            onMenuItemClick={() => onMenuItemClick(NAV_MENU_LABELS.SUPPORT_US)}
+          />
 
-      <Link onClick={onClose} href={`/${ROUTES.DOCS_EN}`} className={cx('category-link', color)}>
-        Docs
-      </Link>
+          <SchoolMenu isVisible={activeDropdowns.has(NAV_MENU_LABELS.SUPPORT_US)}>
+            <SchoolMenu.Item
+              className="support-title"
+              title="Your donations help us cover hosting, domains, licenses, and advertising for courses
+                          and events. Every donation, big or small, helps!"
+              url="#"
+              color={color}
+            />
+            <SchoolMenu.Item
+              className="support-title"
+              title="Thank you for your support!"
+              url="#"
+              color={color}
+            />
+            {donateOptionsForNavMenu.map((option) => (
+              <SchoolMenu.Item
+                key={option.id}
+                icon={option.icon}
+                title={option.linkLabel}
+                url={option.href}
+                color={color}
+                onClick={onClose}
+              />
+            ))}
+          </SchoolMenu>
+        </div>
+      </div>
     </nav>
```

Comment:
as a suggestion above, if you're going to split the main header menu into separate blocks, you should split the mobile menu as well. It came out as a very large component